modules/gitops-release-notes-1-8-0.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-0_{context}"]
+= Release notes for {gitops-title} 1.8.0
+
+{gitops-title} 1.8.0 is now available on {product-title} 4.8, 4.9, 4.10, 4.11, and 4.12.
+
+[id="errata-updates-1-8-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, you can enable workload monitoring for specific Argo CD instances by setting the `.spec.monitoring.enabled` flag value to `true`. As a result, the Operator creates a `PrometheusRule` object that contains alert rules for each Argo CD component. These alert rules trigger the firing of an alert when the replica count of the corresponding component has drifted from the desired state for a certain amount of time. You can modify the rules as per the requirements and prevent the Operator from overwriting them. link:https://issues.redhat.com/browse/GITOPS-2459[GITOPS-2459]
```

Comment:
Instead of `You can modify the rules as per the requirements and prevent the Operator from overwriting them.` can we just say `Changes made to the prometheusRule by the users will not be overwritten by the operator`?

Reply:
https://issues.redhat.com/browse/GITOPS-2457 should be linked instead of https://issues.redhat.com/browse/GITOPS-2459? 
Correct me if I am wrong @jaideepr97

Reply:
sure
i dont know how much it matters either way